__tests__/app.test.js

```diff
@@ -77,3 +77,20 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("GET /api/users", () => {
+  test("200: returns an array of user objects, each with username, name and avatar_url properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((res) => {
```

Comment:
- [ ] Response needs to be sent back as an object with the key of users
- [ ] Need to check the length of the array to check you're getting all the users

Reply:
test altered